aiogram/contrib/fsm_storage/mongo_aiomongo.py

```diff
@@ -92,7 +92,7 @@ async def get_db(self) -> Database:
     @staticmethod
     async def apply_index(db):
         for collection in COLLECTIONS:
-            await db[collection].create_index(keys=[('chat', 1), ('user', 1)],
+            await db[collection].create_index(keys=[('chat_id', 1), ('user_id', 1)],
```

Comment:
This file will be removed, so I'll leave comments only in `mongo_motor.py`. Feel free to delete this file, and move content of `mongo_motor.py` to `mongo.py`